chrome/extension/options.css

```diff
@@ -24,7 +24,7 @@ body {
   line-height: 1.4em;
   background: #f5f5f5;
   color: #4d4d4d;
-  width: 300px;
+  width: 320px;
```

Comment:
I'd approve this PR if it weren't for this change and the change in popup.jade. Let's handle this in a separate PR, at e.g. https://github.com/ChessCom/browser-extension/pull/77. 

The migration from 300 to 320 is a little more complicated than just this. 

![](https://cl.ly/1a0V0f3q0H1v/Image%202017-03-16%20at%208.36.18%20PM.png)